source/features/conversation-activity-filter.tsx

```diff
@@ -99,8 +109,8 @@ async function handleSelection({target}: Event): Promise<void> {
 	);
 
 	// Update the state of the other dropdown
-	select(`.${dropdownClass} [aria-checked="false"][data-value="${currentSetting}"]`)!.setAttribute('aria-checked', 'true');
-	select(`.${dropdownClass} [aria-checked="true"]:not([data-value="${currentSetting}"])`)!.setAttribute('aria-checked', 'false');
+	select(`.${dropdownClass} [aria-checked="false"][data-value="${currentSetting}"]`)?.setAttribute('aria-checked', 'true');
+	select(`.${dropdownClass} [aria-checked="true"]:not([data-value="${currentSetting}"])`)?.setAttribute('aria-checked', 'false');
```

Comment:
in Files Changed tab, sticky header is not supported

Reply:
This is why I don’t want to make this change. We use `!.` because if GitHub changes some classes we will get some errors in the console. `?.` makes these errors silent

Reply:
If the changes required are actually just these few lines, maybe we can just wrap this in a an if !isPRFiles

Reply:
Yes, I' change into it wrapped as if
Thanks!